web-server/views/skills/show.jade

```diff
@@ -1,10 +1,15 @@
 extends ../layout
 
 block content
+  ul.markdown-page-options.list-inline
+    li
+      a(href=sourceUrl target="_blank") View On GitHub
+    li
+      a(href=editeUrl target="_blank") Improve This Page
 
   h1.skill-show-page-heading
```

Comment:
remove this h1 with the checkbox. We're going to need to write some client javascript to dynamically add this checkbox on page load since we cannot easily modify the HTML we get from the markdown file. Ignore that for now and just remove this h1 and input